quetz/rest_models.py

```diff
@@ -117,11 +120,22 @@ class ChannelActionEnum(str, Enum):
 
 
 class ChannelMetadata(BaseModel):
-    includelist: Optional[List[str]] = Field(
+
+    """
+    examples:
+    - includelist: ["numpy", "pandas"]
```

Comment:
The `list` mode is just syntactic sugar so users with a single mirror channel don't need to type a dict, right? I feel like the extra complication might not be worth it in this case, and I would prefer to just have a single way of doing this rather than two every so slightly different ones. What do you thikn?

Reply:
I could also use the list when I mirror multiple channels but still have _one_ list of packages I want to let through or exclude.
Also, it makes this stay backward-compatible 😬